src/starcounter-debug-aid.html

```diff
@@ -173,6 +81,11 @@
                 //this.$.errorDialog.reset();
             },
             openDialog: function () {
+                if (!Polymer.isInstance(this.$.ui)) {
+                    Polymer.Base.importHref("src/starcounter-debug-aid-ui.html", () => {
```

Comment:
Maybe I am missing something about how polyserve works...

Here is what I see:
- if I place `<link rel="import" href="whatever.html" />` it results to `http://127.0.0.1:8000/components/starcounter-debug-aid/src/whatever.html`
- if I do `Polymer.Base.importHref("whatever.html");` it results to `http://127.0.0.1:8000/components/starcounter-debug-aid/whatever.html` **/src** part is missing.